tests/client/amo/components/TestAddonDetail.js

```diff
@@ -452,3 +465,13 @@ describe('AddonDetail', () => {
     });
   });
 });
```

Comment:
Since you missed passing some props to `AddonCompatibilityError` I think it would be good to add a test for that. You could use a stub component and make assertions about what props it received.